README.md

```diff
@@ -1 +1,4 @@
 # node-js-course
+
+## npm install
```

Comment:
All code should be into code blocks.
Use code blocks, please
[code-blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/)

FYI: `#` using via header. [headers](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)

Reply:
Done, thx